tests/playwright/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@podman-desktop/tests-playwright",
+  "version": "1.9.0-next",
+  "description": "Playwright-based testing libraries for Podman Desktop and its extensions",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist",
+    "!dist/specs/*",
+    "!dist/setupFiles/extended-hooks.*"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "rimraf dist/ && vite build",
+    "package": "npm run build && npm link && npm pack",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "publish:next": "yarn publish --registry=https://registry.npmjs.org/ --no-git-tag-version --new-version 0.0.1-\"$(date +%s)\""
+  },
+  "dependencies": {
+    "electron": "29.1.4",
+    "rimraf": "5.0.5",
+    "typescript": "^5.4.2"
```

Comment:
AFAIK these 2 are dev dependencies

Reply:
I don't know then for tests if we should move also electron to a devDependency or if @playwright/test should be moved to dependency

probably all of them should be devDependencies and a peerDependencies should be added on electron and @playwright/test

Reply:
will try. thx